keystorage/aws/src/test/java/tech/pegasys/signers/aws/SecretValueConverterUtilTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.aws;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.signers.aws.SecretValueConverterUtil.mapSecretValue;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+class SecretValueConverterUtilTest {
```

Comment:
Test case for separating lines by carriage return as well? Or do we only want to support \n?

Reply:
since we are using `String.lines()`, it technically support all kind of line termination.